### PR TITLE
fix: [AA-1078] a11y and styling for weekly goals

### PR DESCRIPTION
**Pull request by cdeery**
fix: [AA-1078] a11y and styling for weekly goals

Change radio buttons back to regular buttons.

Fix styling to match the spec

Fix tests

### CODE REVIEW

File: src/course-home/outline-tab/OutlineTab.test.jsx [hunk 1]
```diff
@@ -437,7 +437,7 @@ describe('Outline Tab', () => {
       const spy = jest.spyOn(thunks, 'saveWeeklyLearningGoal');
 
       await fetchAndRender();
-      const button = await screen.getByTestId('weekly-learning-goal-input-regular');
+      const button = await screen.getByTestId('weekly-learning-goal-input-Regular');
```
Comment (MatthewPiatetsky) [inline src/course-home/outline-tab/OutlineTab.test.jsx:440]: curious why this needs to be capitalized?

File: src/course-home/outline-tab/widgets/FlagButton.jsx [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
+  isSelected,
 }) {
   return (
-    <div
-      className="flag-button col flex-grow-1 p-3"
+    <button
+      type="button"
+      className={classnames('flag-button m-1.5 pt-3 pb-md-3 pb-xl-0',
+        isSelected ? 'flag-button-selected' : '')}
+      // aria-checked={isSelected}
```
Comment (MatthewPiatetsky) [inline src/course-home/outline-tab/widgets/FlagButton.jsx:17]: why are these comments here?

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 3]
```diff
@@ -47,16 +48,26 @@ function WeeklyLearningGoalCard({
...
+        data-testid="weekly-learning-goal-card"
+      >
         <Card.Body className="p-3.5">
           <Card.Title>
             <h2 className="h4 m-0 text-primary-500">{intl.formatMessage(messages.setWeeklyGoal)}</h2>
           </Card.Title>
-          <Card.Text className="text-gray-700">
+          <Card.Text
+            className="text-gray-700"
+              // id="learning_goal_choices_label"
```
Comment (MatthewPiatetsky) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx:61]: should this comment be here?

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 4]
```diff
@@ -47,16 +48,26 @@ function WeeklyLearningGoalCard({
...
+          >
             {intl.formatMessage(messages.setWeeklyGoalDetail)}
           </Card.Text>
           <div
-            className="row w-100 m-0 p-0 justify-content-between"
+            className="flag-button-container m-0 p-0"
+              // making this work as a radio button for keyboard nav
```
Comment (MatthewPiatetsky) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx:67]: should this comment be here?

**Review by MatthewPiatetsky — APPROVED** [review]
if you squash your commits maybe it won't complain since you have one commit without fix

i'd remove unnecessary comments

not sure without screenshots if the things I pointed out are fixed, but rather than spending time making screenshot and gifs for each case i'm fine just reviewing it on prod since its behind a flag anyways and this will keep the iteration cycle faster

File: src/course-home/outline-tab/OutlineTab.test.jsx (hunk 1 above)
Comment (cdeery) [reply src/course-home/outline-tab/OutlineTab.test.jsx:440]: The title is now the actual title, which is capitalized

File: src/course-home/outline-tab/widgets/FlagButton.jsx (hunk 2 above)
Comment (cdeery) [reply src/course-home/outline-tab/widgets/FlagButton.jsx:17]: There are still some a11y issues, and I'm not quite done trying to make it act like a radio button.

**Pull request merged**